packages/BuildTranslations/src/cli.js

```diff
@@ -0,0 +1,27 @@
+import arg from "arg";
+import buildTranslations from "./index";
+
+function parseArgumentsIntoOptions(rawArgs) {
+  const args = arg(
+    {
+      "--out-dir": String,
+      "--yaml-file-ext": String,
+    },
+    {
+      argv: rawArgs,
+    }
+  );
+
+  const sourcePath = args._[2];
+
+  return {
+    sourcePath,
+    outputPath: args["--out-dir"],
+    yamlFileExtension: args["--yaml-file-ext"],
+  };
+}
+
+export async function cli(args) {
+  const options = parseArgumentsIntoOptions(args);
+  await buildTranslations(options);
```

Comment:
does it worth adding a message at the end? `console.log("locales have been processed")`